Update naming and images for stack trace and event navigator (#9043)
getsentry/sentry-docs: docs/product/issues/issue-details/error-issues/index.mdx

![Issue details](../issue_details_page.png)

> While the <SandboxLink scenario="oneIssue" projectSlug="react">Issue Details</SandboxLink> page displays information about a specific error event that's part of an issue, you can navigate between the events of an issue using the event navigator. Subscribe to an error issue by clicking the bell icon and fine tune [workflow notifications](/product/alerts/notifications/#workflow-notifications) related to the issue in **User Settings > Notifications**. Just below the issue title, the event description is displayed along with an icon representing the error level of the event:

Reviewer: ```suggestion
You can view information about a specific error event that's part of an issue on the <SandboxLink scenario="oneIssue" projectSlug="react">Issue Details</SandboxLink> page. You can also navigate between the events of an issue using the event navigator. To subscribe to an error issue and receive alerts about it, click the bell icon, then fine tune [workflow notifications](/product/alerts/notifications/#workflow-notifications) related to the issue in **User Settings > Notifications**. The event description is displayed just below the issue title along with an icon representing the error level of the event:
```
